packages/common/src/store/playback-position/slice.ts

```diff
@@ -10,16 +10,16 @@ type InitializePlaybackPositionStatePayload = {
 
 type SetTrackPositionPayload = {
   trackId: ID
+  userId?: ID | null
```

Comment:
should these accept empty? seems like it's a no-op if we don't have user id, so maybe just don't fire the action in the first place? idk which is better

Reply:
This made it easier for type and not having to check it user existed everywhere this was called if i remember correctly, but I agree it's a bit weird